Source/CronosPlayUnreal/Public/DefiWalletCoreActor.h

```diff
@@ -997,6 +1006,24 @@ class CRONOSPLAYUNREAL_API ADefiWalletCoreActor : public AActor {
                         FString approvedAddress, bool approved,
                         FErc1155ApproveDelegate Out);
 
+    /**
+     * send modifying contract call
+     * @param contractAddress contract address
+     * @param walletindex wallet index which starts from 0
+     * @param abiJson abi json
+     * @param functionName function name
+     * @param functionArgs function args injson
```

Comment:
for the function args, it'll be good to give an example, because the format is not obvious just from the description... maybe using doxygen's `\code ... \endcode` command?